### ISSUE TITLE
sourcehut/gitsrht: Creation of source tarballs fails due to missing gzip

### ISSUE THREAD

**Issue by malte-v** [NixOS/nixpkgs#138511]
### Describe the bug
When downloading a source tarball from my NixOS-powered sourcehut instance, I get an empty file and the following message in gitsrht's log:
```
Sep 19 11:02:55 servant gunicorn[46430]: gzip -cn: gzip: command not found
```

### Notify maintainers
@eadwu @tomberek

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"aarch64-linux"`
 - host os: `Linux 5.10.17, NixOS, 21.11.20210605.883c097 (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre20210601_5985b8b`
 - channels(root): `"nixos, home-manager"`
 - channels(malte): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by malte-v** [comment]
Will be fixed by #133984.

**Comment by nessdoor** [comment]
The tests introduced in #271324 revealed that this issue seems resolved in the current version of Sourcehut.
